src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -44,6 +46,10 @@ def dependency(self):
     def path(self):
         return self._dependency.path
 
+    def install_library(self, library: str) -> list[DependencyProblem]:
```

Comment:
@ericvergnaud : Do we want to add this to the graph? We could also add it to the path lookup as we are not doing anything with the graph.